tests/manage/z_cluster/cluster_expansion/test_node_expansion.py

```diff
@@ -57,5 +57,21 @@ def test_add_node(self):
                 status=constants.NODE_READY
             )
         else:
-            pytest.skip("UPI not yet supported")
+            new_nodes = 3
```

Comment:
for add node, I'm thinkingto have a separate function for aws and vmware since aws has many options to add.

Reply:
https://github.com/red-hat-storage/ocs-ci/pull/1847 --> for AWS.
like that, separate function for vmware, like  test_add_node_vnware()

Reply:
As discussed, test case part is removed fro this PR